homeassistant/components/device_tracker/tplink.py

```diff
@@ -361,50 +363,49 @@ def _update_info(self):
         base_url = 'http://{}'.format(self.host)
 
         header = {
-            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12;"
-                          " rv:53.0) Gecko/20100101 Firefox/53.0",
-            "Accept": "application/json, text/javascript, */*; q=0.01",
-            "Accept-Language": "Accept-Language: en-US,en;q=0.5",
-            "Accept-Encoding": "gzip, deflate",
-            "Content-Type": "application/x-www-form-urlencoded; "
-                            "charset=UTF-8",
-            "X-Requested-With": "XMLHttpRequest",
-            "Referer": "http://" + self.host + "/",
-            "Connection": "keep-alive",
-            "Pragma": "no-cache",
-            "Cache-Control": "no-cache"
+            USER_AGENT:
+                "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12;"
+                " rv:53.0) Gecko/20100101 Firefox/53.0",
+            ACCEPT: "application/json, text/javascript, */*; q=0.01",
+            ACCEPT_LANGUAGE: "Accept-Language: en-US,en;q=0.5",
+            ACCEPT_ENCODING: "gzip, deflate",
+            CONTENT_TYPE: "application/x-www-form-urlencoded; charset=UTF-8",
+            HTTP_HEADER_X_REQUESTED_WITH: "XMLHttpRequest",
+            REFERER: "http://{}/".format(self.host),
+            CONNECTION: KEEP_ALIVE,
```

Comment:
There is no reason to move these to constants. They can just remain strings. All constants that are in `const.py` will be loaded into memory of every HASS instance so only common constants should make it into that file. So let's keep things like X_REQUESTED_WITH, KEEP_ALIVE and NO_CACHE in this file.

Reply:
Sorry, I don't understand. Beside `HTTP_HEADER_X_REQUESTED_WITH` and `NO_CACHE` which are in `const.py` is the rest coming out of `aiohttp.hdrs`.

Reply:
Yeah, I meant that we shouldn't add any HTTP headers to `const.py` anymore.